vividus-plugin-excel/src/main/java/org/vividus/bdd/transformer/ExcelTableTransformer.java

```diff
@@ -94,17 +118,33 @@ public String transform(String tableAsString, TableParsers tableParsers, TablePr
         return data;
     }
 
-    private List<String> extractDataFromAddresses(IExcelSheetParser sheetParser, String addresses)
+    private List<List<String>> transformDataTable(List<String> headers, List<String> data)
```

Comment:
you shouldn't transform linear data (list) to table, you should extract data as table from excel (without extra transformation)